index.js

```diff
@@ -234,9 +234,9 @@
 		},
 
 		sumCols: function () {
-			const columnIds = this.$headerCols.map(($column) => {
+			const columnIds = this.$headerCols.map(($column, idx) => {
 				if (this.isBoardTypeNextGen) {
-					return $column.dataset.rbdDragHandleDraggableId;
+					return idx;
```

Comment:
so there is no information in the html about which column it is anymore, but just walking them in order and using their position as "id" works fine.